src/Autosuggest.js

```diff
@@ -274,10 +280,19 @@ class Autosuggest extends Component {
       }
     };
     const onMouseEnter = (event, { sectionIndex, itemIndex }) => {
-      updateFocusedSuggestion(sectionIndex, itemIndex);
+      if (!focusedUsingKeyboard) {
+        updateFocusedSuggestion(sectionIndex, itemIndex);
+      }
     };
     const onMouseLeave = () => {
-      updateFocusedSuggestion(null, null);
+      if (!focusedUsingKeyboard) {
+        updateFocusedSuggestion(null, null);
+      }
+    };
+    const onMouseMove = () => {
+      if (focusedUsingKeyboard) {
+        updateFocusedUsingKeyboard(false);
+      }
```

Comment:
I tried something like this earlier and had trouble with trhe enter/leave events arriving after the timeout. I could try it with a longer delay 

Another option might be to add the mouysemove handler only when focus has been set with the keyboard and remove the handler the firsst time it is called.

I'll have a go with these and update the PR if I get one of them to work :-)